python/feathub/processors/flink/table_builder/tests/test_flink_table_builder_native_flink_sliding_window.py

```diff
@@ -0,0 +1,1040 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from datetime import timedelta
+from math import sqrt
+from typing import Dict
+
+import pandas as pd
+from feathub.feature_views.transforms.python_udf_transform import PythonUdfTransform
+
+from feathub.common.types import Int64, String, Float64, MapType
+from feathub.feature_views.derived_feature_view import DerivedFeatureView
+from feathub.feature_views.feature import Feature
+from feathub.feature_views.sliding_feature_view import (
+    SlidingFeatureView,
+    ENABLE_EMPTY_WINDOW_OUTPUT_CONFIG,
+    ENABLE_STATIC_WINDOW_OUTPUT_CONFIG,
+)
+from feathub.feature_views.transforms.sliding_window_transform import (
+    SlidingWindowTransform,
+)
+from feathub.processors.flink.flink_table import flink_table_to_pandas
+from feathub.processors.flink.table_builder.tests.sliding_window_test_base import (
+    SlidingWindowTestBase,
+)
+from feathub.table.schema import Schema
+
+
+class FlinkTableBuilderNativeFlinkSlidingWindowTransformTest(SlidingWindowTestBase):
```

Comment:
Can you add python doc explaining the difference between FlinkTableBuilderNativeFlinkSlidingWindowTransformTest and FlinkTableBuilderSlidingWindowTransformTest?